modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -29,8 +29,9 @@
         :disabled="!!relationshipErrors"
         @click="saveRelationship"
       />
+      <!-- Currently we cannot create a document if we can't edit them -->
       <AposButton
-        v-else-if="moduleOptions.canEdit && moduleOptions.showCreate"
+        v-else-if="moduleOptions.canCreate && moduleOptions.canEdit && moduleOptions.showCreate"
```

Comment:
This button has to be available if you `canCreate`, even if you can't edit. Otherwise we are unable to take advantage of our create permission, right?

Reply:
(That scenario does not arise in core but it does arise in AP so this logic needs to be looking for canCreate and not canEdit)

Reply:
Yes but currently we cannot create a document if we cannot edit this one.. so there is a missing part somewhere in the core I suppose.

Reply:
I remove the  `canEdit` here but we'll need to fix the fact that we cannot create a piece without the edit permission.